### PR TITLE
add recipes form Galaxy-ML

### PR DESCRIPTION
**Pull request by qiagu**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by qiagu** [comment]
@bioconda/core

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Comment by dpryan79** [comment]
Yay Galaxy ML tooling!

**Comment by dpryan79** [comment]
@bioconda-bot recheck

File: recipes/galaxy-ml/meta.yaml [hunk 1]
```diff
@@ -11,8 +11,7 @@ source:
 
 build:
   number: 0
-  skip: True
```
Comment (qiagu) [inline recipes/galaxy-ml/meta.yaml]: I'm new to bioconda recipes, but this one failed previously without `skip: True`. @dpryan79

Comment (dpryan79) [reply recipes/galaxy-ml/meta.yaml]: With `skip: True` you're telling the build system to not build the package.

Comment (dpryan79) [reply recipes/galaxy-ml/meta.yaml]: Are you at GCC? If so, please remind me to have a look at this PR tomorrow so we can get it building and made available.

Comment (dpryan79) [reply recipes/galaxy-ml/meta.yaml]: Or if you're in Jeremy's group just ask him to remind me :)

Comment (qiagu) [reply recipes/galaxy-ml/meta.yaml]: I'm not, unfortunately. I can ping you though if the problem is still there.

Comment (qiagu) [reply recipes/galaxy-ml/meta.yaml]: Sounds cool.

**Comment by dpryan79** [comment]
I'll try to get pytabix building on OSX. Worst case we can just skip there, since I don't know of any Galaxy instances running on OSX.

**Comment by dpryan79** [comment]
This should (hopefully) now build on OSX too. Please note that the biocontainer for this may or may not correctly work. We can't test tensorflow stuff because it's linked against an incompatible libc version. So if possible, please try to test that will a local install (galaxy or otherwise) and see if it works.

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Comment by qiagu** [comment]
Glad we finally made it. Thank you very much! @dpryan79

**Pull request merged**